packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/OutputTypes/ScmKnownParametersTests.cs

```diff
@@ -8,16 +8,6 @@ namespace Microsoft.Generator.CSharp.ClientModel.Tests.OutputTypes
 {
     internal class ScmKnownParametersTests
     {
-        [Test]
-        public void TestTokenAuth()
-        {
-            MockHelpers.LoadMockPlugin(keyCredentialType: () => typeof(int));
-
-            var result = ClientModelPlugin.Instance.TypeFactory.KeyCredentialType;
-            Assert.IsNotNull(result);
-            Assert.AreEqual(new CSharpType(typeof(int)), result);
-        }
```

Comment:
this test is removed because the property this is testing has been removed.
The functionality of this is tested somewhere else now.